fastlane/lib/fastlane/actions/upload_app_privacy_details_to_app_store.rb

```diff
@@ -0,0 +1,285 @@
+                                       description: "The bundle identifier of your app",
+                                       code_gen_sensitive: true,
+                                       default_value: CredentialsManager::AppfileConfig.try_fetch_value(:app_identifier),
+                                       default_value_dynamic: true),
+          FastlaneCore::ConfigItem.new(key: :team_id,
+                                       env_name: "FASTLANE_ITC_TEAM_ID",
+                                       description: "The ID of your App Store Connect team if you're in multiple teams",
+                                       optional: true,
+                                       is_string: false, # as we also allow integers, which we convert to strings anyway
+                                       code_gen_sensitive: true,
+                                       default_value: CredentialsManager::AppfileConfig.try_fetch_value(:itc_team_id),
+                                       default_value_dynamic: true),
+          FastlaneCore::ConfigItem.new(key: :team_name,
+                                       env_name: "FASTLANE_ITC_TEAM_NAME",
+                                       description: "The name of your App Store Connect team if you're in multiple teams",
+                                       optional: true,
+                                       code_gen_sensitive: true,
+                                       default_value: CredentialsManager::AppfileConfig.try_fetch_value(:itc_team_name),
+                                       default_value_dynamic: true),
+
+          # JSON paths
+          FastlaneCore::ConfigItem.new(key: :json_path,
+                                       env_name: "UPLOAD_APP_PRIVACY_DETAILS_TO_APP_STORE_JSON_PATH",
+                                       description: "Path to the app usage data JSON path",
+                                       is_string: true,
+                                       optional: true,
+                                       verify_block: proc do |value|
+                                         UI.user_error!("Could not find JSON file at path '#{File.expand_path(value)}'") unless File.exist?(value)
+                                         UI.user_error!("'#{value}' doesn't seem to be a JSON file") unless FastlaneCore::Helper.json_file?(File.expand_path(value))
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :output_json_path,
+                                       env_name: "UPLOAD_APP_PRIVACY_DETAILS_TO_APP_STORE_OUTPUT_JSON_PATH",
+                                       description: "Path to the app usage data JSON file generated by interactive questions",
+                                       default_value: File.join(DEFAULT_PATH, DEFAULT_FILE_NAME)),
+
+          # Skipping options
+          FastlaneCore::ConfigItem.new(key: :skip_json_file_saving,
```

Comment:
nit: Should this set the conflict options? I think if `output_json_path` is set this option doesn't make much sense